trocr/evaluation-dataset/handwritten-typed-text-classification/research.md

```diff
@@ -0,0 +1,26 @@
+# [Research] TrOCR Encoder + FFN Decoder
+
+#### Overview
+To create a robust classification model for our task, multiple Convolutional Neural Network (CNN) models were explored and assessed. Details of each attempted model, along with their respective implementations, can be accessed in the [Introduction section](https://github.com/BU-Spark/ml-herbarium/blob/dev/trocr/evaluation-dataset/handwritten-typed-text-classification/notebooks/Classifier_NN.ipynb) of the project's Jupyter Notebook.
```

Comment:
"for our task" -- There is no description of what the task is. I suggest starting the readme with the problem statement.

Reply:
Oh. I see there is a readme. Would be good to link the readme and explain what this doc is versus the readme. Presumably it is notes from your research this summer.